### PR TITLE
Adds react-router, bootswatch formatting, Main page with Nav and Foot components and Login and Dashboard Routes

### PR DESCRIPTION
**Pull request by 08martinm**
Adds:
- react-router
- bootswatch formatting
- Main Route
- Login Route
- Dashboard Route
- Nav component
- Foot component

### CODE REVIEW

File: client/src/pages/components/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            <Link className="navbar-brand" to="/login">EtherList</Link>
+          </div>
+
+          <div className="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul className="nav navbar-nav">
+              <li className="active"><Link to="/login">Log In<span className="sr-only">(current)</span></Link></li>
+              <li><Link to="/home">Dashboard</Link></li>
+            </ul>
+            <ul className="nav navbar-nav navbar-right">
+              <li><Link to="/login">Sign Out</Link></li>
```
Comment (donovanCarreno) [inline client/src/pages/components/Nav.jsx:29]: should this be: "<Link to="/logout">"...?

File: client/src/pages/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                <button className='signUp btn btn-danger'>Sign Up!</button>
+              </div>
+
+            </fieldset>
+          </form>
+        </div>
+      </div>
+    )
+  }
+}
```
Comment (donovanCarreno) [inline client/src/pages/Login.jsx:42]: the onClick & onChange methods are currently just placeholders, right?  We haven't written them yet?

**Comment by donovanCarreno** [comment]
🚢

**Comment by cassiebaer** [comment]
:ship:

**Pull request merged**